test/utils/fizzy_engine.cpp

```diff
@@ -72,9 +73,54 @@ bytes_view FizzyEngine::get_memory() const
     return {m_instance->memory->data(), m_instance->memory->size()};
 }
 
-std::optional<WasmEngine::FuncRef> FizzyEngine::find_function(std::string_view name) const
+namespace
 {
-    return fizzy::find_exported_function(m_instance->module, name);
+FuncType translate_signature(std::string_view signature)
+{
+    std::string_view inputs(signature.begin(), signature.find(":"));
+    std::string_view outputs(inputs.end() + 1);  // Is this a terrible hack?
+
+    FuncType func_type;
+    for (auto input : inputs)
+    {
+        if (input == 'i')
+            func_type.inputs.push_back(fizzy::ValType::i32);
+        else if (input == 'I')
+            func_type.inputs.push_back(fizzy::ValType::i64);
+        else
+            throw std::runtime_error{"invalid type"};
+    }
+    for (auto output : outputs)
+    {
+        if (output == 'i')
+            func_type.outputs.push_back(fizzy::ValType::i32);
+        else if (output == 'I')
+            func_type.outputs.push_back(fizzy::ValType::i64);
+        else
+            throw std::runtime_error{"invalid type"};
+    }
+
+    return func_type;
+}
+
+inline bool operator!=(const FuncType& lhs, const FuncType& rhs)
```

Comment:
This should go to `types.hpp` or other common header.
Porbably with `operator==`, too, and unit tests.

Reply:
Done in #360.